### ISSUE TITLE
"bandage" function missing

### ISSUE THREAD

**Issue by ngyoung** [jsebold666/RazorEnhanced#14]
```
if not bandage
        headmsg 'No Bandages!' 32
        stop
 endif
```
the function "bandage' doesn't show up in documentation but is an accepted function in the UOS editor.  I believe it is just a copy of bandageself.  If run by itself it will search for a bandage and use it on yourself.  When put into an if condition like above, if there is no bandages in your inventory it will trigger the condition.  Conversely, when running:
```
if bandage
    headmsg 'Bandaging' 32
    stop
endif
```
it will find a bandage and use it on self.  Replacing it with the function "bandageself" performs the exact same way.

**Comment by ngyoung** [comment]
Just to follow up.  Changing to "if not bandageself" in RE throws an error.  I don't think it was setup under the covers to be able to be used to check if you have bandages like it does in UOS
[12:14:09] - Exception: Error:	Unknown expression
[12:14:09] - Type:	OPERAND
[12:14:09] - Word:	bandageself
[12:14:09] - Line:	83
[12:14:09] - Code:	    if not bandageself

**Comment by credzba** [comment]
Resolved in next build  8.0.5 or 7.7. 23
